apps/emqx_authn/i18n/emqx_authn_api_i18n.conf

```
@@ -204,24 +204,13 @@ emqx_authn_api {
     }
   }
 
-  like_username {
+  like_user_id {
     desc {
-      en: """Fuzzy search username."""
+      en: """Fuzzy search user_id (username or clientid)."""
       zh: """使用用户名模糊查询。"""
     }
     label {
-      en: """like_username"""
-      zh: """模糊用户名"""
-    }
-  }
-
-  like_clientid {
-    desc {
-      en: """Fuzzy search clientid."""
-      zh: """使用客户端标识符模糊查询。"""
-    }
-    label {
-      en: """like_clientid"""
+      en: """like_user_id"""
       zh: """模糊用户名"""
```

I don't known how to say in Chinese🙃 Just using `like_user_id` ?
```suggestion
      zh: """like_user_id"""
```